packages/create-remix/templates/arc-stack/package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "private": true,
+  "name": "remix-app-template",
+  "description": "",
+  "license": "",
+  "scripts": {
+    "postinstall": "remix setup node",
+    "build": "remix build",
+    "dev:remix": "remix watch",
+    "dev:arc": "arc sandbox",
+    "dev": "cross-env ARC_LOCAL=1 ARC_TABLES_PORT=5555 run-p dev:*"
+  },
+  "dependencies": {
+    "@architect/functions": "5.0.0-RC.1",
```

Comment:
IMO it would be safer to use `4.1.2` of architect/functions since a user's global Architect installation (recommended in the Readme here) is v9.

The latest arc/functions@RCs are for the Arc 10 RC.